main/linux-vanilla/APKBUILD

```diff
@@ -12,7 +12,7 @@ pkgdesc="Linux vanilla kernel"
 url="http://kernel.org"
 depends="mkinitfs"
 _depends_dev="perl gmp-dev elfutils-dev bash"
-makedepends="$_depends_dev sed installkernel bc linux-headers linux-firmware"
+makedepends="$_depends_dev flex bison sed installkernel bc linux-headers linux-firmware openssl-dev"
```

Comment:
we currently use libressl, so if this is really needed, then it should be `libressl-dev`.

I wonder why we don't hit this issue on our builders?